__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const request = require("supertest")
+const app = require("../app.js")
+const db = require ("../db/connection.js")
+const seed = require("../db/seeds/seed.js")
+const data = require("../db/data/test-data")
+
+beforeEach(()=>{
+    return seed(data)
+})
+
+afterAll(()=>{
+    db.end()
+})
+
+describe("all bad URLs",()=>{
+    describe("/api/topics",()=>{
+        test("404 URL NOT FOUND", () => {
+        return request(app)
+        .get("/api/tipics")
+        .expect(404)
+        .then(({ body }) => {
+        expect(body.msg).toBe("URL NOT FOUND")
+        }) 
+     })
+    })
+})
```

Comment:
This error is not specific to any one endpoint, but rather a single test for any time a user makes a request to an endpoint that doesn't exist in your API

_( now that you have done it once, you don't need to write this again for each endpoint )_